app/uk/gov/hmrc/merchandiseinbaggage/repositories/DeclarationJourneyRepository.scala

```diff
@@ -56,6 +56,8 @@ class DeclarationJourneyRepository @Inject()(mongo: () => DB, @Named("declaratio
     find(query).map(_.headOption)
   }
 
+  def findAllDeclarations(): Future[List[DeclarationJourney]] = findAll()
```

Comment:
if this is only meant for testing, why can't we just use the `findAll()` directly in the tests ?